### PR TITLE
Create 91-ipfs-dev-meetings-2018.md

### PR DESCRIPTION
**Pull request by momack2**
Here's a rough draft at a blog post for our dev meeting videos. Would love review from @doctorrobinson and @daviddias

### CODE REVIEW

File: content/post/91-ipfs-dev-meetings-2018.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+- [Lightning Demos - Textile - Andrew Hill](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=32)
+- [Lightning Demos - pServer - Wei Yang](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=33)
+- [Lightning Demos - window.IPFS - Alan Shaw](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=34)
+- [Lightning Demos - IPLD Explorer CLI - Alan Shaw](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=35)
+- [Lightning Demos - Files and IPFS Companion - Marcin Rataj](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=36)
+- [Lightning Demos - VRQ Recovery - jonnycrunch](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=37)
+- [Project Updates, Lightning Talks Q&A - Decentralized Identifiers - jonnycrunch](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=38)
+- [Live Keynotes - Peer Discovery Protocols - Juan Benet](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=39)
+
+
```
Comment (daviddias) [inline content/post/91-ipfs-dev-meetings-2018.md]: Are there urls to the Posters and Deep Dives presentations?

File: content/post/91-ipfs-dev-meetings-2018.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+![](https://media.giphy.com/media/iCivRRnqS1SsadMjme/giphy.gif)
+
+The week was great for core developers, contributors and researchers to spend time together diving deep on open design problems, demoing exciting new tools and products being built on or around IPFS and Libp2p, and exploring the path forward for the protocols and working groups. Check out the [jam-packed schedule](https://developersmeetingsberlin2018.sched.com/) to see what we discussed.
+
+Much of the rich knowledge was shared in lightning talks and presentations that we’re excited to now share with the world! Learn about the Projects’ vision, get updated on what the projects were working on in mid 2018, and explore the ecosystem of demos, tools, apps, and exciting improvements to the core protocol in [this YouTube playlist](https://www.youtube.com/watch?v=ctjvfX1EoWY&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr). You can also see the full list below.
```
Comment (daviddias) [inline content/post/91-ipfs-dev-meetings-2018.md]: On the note of pointing folks to the playlist, I wonder if organizing the talks list at https://github.com/ipfs/developer-meetings/ would make it more friendly to pick what to watch.

Perhaps we can c&p the list below to the repo as well?

File: content/post/91-ipfs-dev-meetings-2018.md (hunk 1 above)
Comment (momack2) [reply content/post/91-ipfs-dev-meetings-2018.md]: Not yet I don't think?

File: content/post/91-ipfs-dev-meetings-2018.md (hunk 2 above)
Comment (momack2) [reply content/post/91-ipfs-dev-meetings-2018.md]: sure - took a pass: https://github.com/ipfs/developer-meetings/pull/48

File: content/post/91-ipfs-dev-meetings-2018.md (hunk 1 above)
Comment (doctorrobinson) [reply content/post/91-ipfs-dev-meetings-2018.md]: I will update these all today

Comment (doctorrobinson) [reply content/post/91-ipfs-dev-meetings-2018.md]: Heads up that this is a bit more work than I originally accounted for and it'll be late tomorrow morning before I finish this up.

Comment (momack2) [reply content/post/91-ipfs-dev-meetings-2018.md]: Does this need to happen before I publish?

Comment (daviddias) [reply content/post/91-ipfs-dev-meetings-2018.md]: Not necessarily. We can then publish a post just on the Poster Sessions and Deep Dives :)

File: content/post/91-ipfs-dev-meetings-2018.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+- [Lightning Demos - Textile - Andrew Hill](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=32)
+- [Lightning Demos - pServer - Wei Yang](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=33)
+- [Lightning Demos - window.IPFS - Alan Shaw](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=34)
+- [Lightning Demos - IPLD Explorer CLI - Alan Shaw](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=35)
+- [Lightning Demos - Files and IPFS Companion - Marcin Rataj](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=36)
+- [Lightning Demos - VRQ Recovery - jonnycrunch](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=37)
+- [Project Updates, Lightning Talks Q&A - Decentralized Identifiers - jonnycrunch](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=38)
+- [Live Keynotes - Peer Discovery Protocols - Juan Benet](https://www.youtube.com/watch?v=Ggdz-u0Fy1g&list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr&index=39)
```
Comment (daviddias) [inline content/post/91-ipfs-dev-meetings-2018.md]: Something is weird with these urls. I just tried to open all of them in Firefox, Firefox Incognito and Chrome and they always take me to Hector's talk, although I can see that the Index is different. Does anyone have the same issue?

Comment (daviddias) [reply content/post/91-ipfs-dev-meetings-2018.md]: @momack2 seems that something is weird with the urls. @olizilla and I are directed to Hector's talk for all of them.

Comment (doctorrobinson) [reply content/post/91-ipfs-dev-meetings-2018.md]: Same issue here. Let me troubleshoot - I'm cleaning up all of the dev meetings videos on the backend today and will find a solution.

Comment (momack2) [reply content/post/91-ipfs-dev-meetings-2018.md]: These used to work for me - but also break now (I just see Matt's talk over and over). Zak - I think it honestly just has to grab these full URLs from the playlist (https://www.youtube.com/playlist?list=PLuhRWgmPaHtRT7ZLH_NmdfJhjubdOiTVr)

File: content/post/91-ipfs-dev-meetings-2018.md (hunk 1 above)
Comment (momack2) [reply content/post/91-ipfs-dev-meetings-2018.md]: yeah - sounds just worth correcting the title. I'll commit these into the post and do a final check to hopefully go out tomorrow

**Pull request merged**